velox/dwio/common/tests/utils/FilterGenerator.h

```diff
@@ -71,8 +71,8 @@ class AbstractColumnStats {
   // ASCII string greater than test data values. Used for row group skipping
   // tests.
   static constexpr const char* kMaxString = "~~~~~";
-  AbstractColumnStats(TypePtr type, RowTypePtr rootType)
-      : type_(type), rootType_(rootType) {}
+  AbstractColumnStats(TypePtr type, RowTypePtr rootType, bool allowNulls = true)
```

Comment:
allowNulls is a property of filters, not ColumnStats. I think it would be cleaner to add allowNulls to FilterSpec, and pass it in through ColumnStats::filter(FilterSpec& spec) - > makeRangeFilter(spec)